test/test_sensors.py

```diff
@@ -156,3 +152,34 @@ def test_collisions():
             prev_collisions = collisions
 
     env.close()
+
+
+def test_static_pointgoal_sensor():
+    config = get_config()
+    if not os.path.exists(config.SIMULATOR.SCENE):
+        pytest.skip("Please download Habitat test data to data folder.")
+    config.defrost()
+    config.TASK.SENSORS = ["STATIC_POINTGOAL_SENSOR"]
+    config.freeze()
+    env = habitat.Env(config=config, dataset=None)
+    random.seed(123)
+    np.random.seed(123)
+
+    env.episodes = [
+        NavigationEpisode(
+            episode_id="0",
+            scene_id=config.SIMULATOR.SCENE,
+            start_position=[-1.3731, 0.08431, 8.60692],
+            start_rotation=[0, -np.pi, 0, np.pi],
+            goals=[NavigationGoal([-1.2731, 0.08431, 8.60692])],
```

Comment:
To make test easy to read let's create `expected_static_pointgoal=[0, 0, -0.1]` and set goal as `start_position  + expected_static_pointgoal`.